core/src/main/java/org/wildfly/plugin/core/GalleonUtils.java

```diff
@@ -149,7 +149,15 @@ public static ProvisioningConfig buildConfig(ProvisioningManager pm,
                 String coords = getMavenCoords(fp);
                 fpl = FeaturePackLocation.fromString(coords);
             } else {
-                fpl = FeaturePackLocation.fromString(fp.getLocation());
+                //Special case for G:A that conflicts with producer:channel that we can't have in the plugin.
```

Comment:
@jfdenise this is a change for your PoC and I'm not sure if it's still valid. Could you please check?

Reply:
Yes, that is needed to not require to end the G:A with a ":".